examples/amp-story/amp-story-shopping.html

```diff
@@ -143,6 +155,50 @@
         </amp-story-grid-layer>
         <amp-story-shopping-attachment theme="dark"></amp-story-shopping-attachment>
       </amp-story-page>
+      <amp-story-page id="inline-light-theme" dir="rtl">
+        <!--
+          Example of Inline config. Multiple product tags with icon defined, right to left language direction.
+          In order for this to work as intended, you must specify dir="rtl" in the <html> tag.
```

Comment:
We might be able to update this comment to just be "RTL example", since the other concepts are explained in previous examples and RTL is independent of those concepts.